pkg/runctl/controller_test.go

```
@@ -269,8 +270,9 @@ func resource(resource string) schema.GroupResource {
 // Return nil if not found.
 func getPipelineRun(name string, namespace string, cf *fake.ClientFactory) k8s.PipelineRun {
 	key := fake.ObjectKey(name, namespace)
-	pipelineRun, _ := k8s.NewPipelineRunFetcher(cf).ByKey(key)
-	return pipelineRun
+	fetcher := k8s.NewPipelineRunFetcher(cf)
+	pipelineRun, _ := fetcher.ByKey(key)
```

Do not ignore errors.